### DISCUSSION TITLE
IncludeMembers does not work if ForAllMembers is configured.

### DISCUSSION THREAD

**Discussion by sergeybannikov** [LuckyPennySoftware/AutoMapper#3749] [NA]
var config = new MapperConfiguration(cfg =>
            {
                //---------------------------------------------------------------
                // With this code destination.Description is null
                cfg.ForAllMaps((obj, cnfg) =>
                    cnfg.ForAllMembers(o
                        => o.Condition((src, dest, value)
                            => true)));
                //---------------------------------------------------------------
                cfg.CreateMap<Source, Destination>()
                    .ForMember(dest => dest.a,
                        opt => opt.MapFrom(src => src.InnerSource.Description))
                    .IncludeMembers(s => s.InnerSource, s => s.OtherInnerSource);
                cfg.CreateMap<InnerSource, Destination>(MemberList.None);
                cfg.CreateMap<OtherInnerSource, Destination>(MemberList.None);
            });

            var mapper = config.CreateMapper();

            var source = new Source { Name = "name", InnerSource = new InnerSource { Description = "description" }, OtherInnerSource = new OtherInnerSource { Title = "title" } };
            var destination = mapper.Map<Destination>(source);
            destination.Name.Should().Be("name");
            destination.Description.Should().Be("description");
            destination.Title.Should().Be("title");
        }
    }

    class Source
    {
        public string Name { get; set; }
        public InnerSource InnerSource { get; set; }
        public OtherInnerSource OtherInnerSource { get; set; }
    }
    class InnerSource
    {
        public string Name { get; set; }
        public string Description { get; set; }
    }
    class OtherInnerSource
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Title { get; set; }
    }
    class Destination
    {
        public string a { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Title { get; set; }
    }

**Comment by lbargaoanu** [comment]
https://github.com/AutoMapper/AutoMapper/discussions/3594
